src/main/vcp_hal/usbd_cdc_interface.c

```diff
@@ -187,6 +195,12 @@ static int8_t CDC_Itf_Control (uint8_t cmd, uint8_t* pbuf, uint16_t length)
     LineCoding.format     = pbuf[4];
     LineCoding.paritytype = pbuf[5];
     LineCoding.datatype   = pbuf[6];
+    // If a callback is provided, tell the upper driver of changes in baud rate
+	 if (pbuf && (length == sizeof (LineCoding))) {
```

Comment:
`pbuf` was already dereferenced, too late to check it ...

Reply:
I'll move the check to cover the read of pbuf.